Makefile

```diff
@@ -773,23 +775,176 @@ out/36-new-york/state.gpkg: data/36-new-york/statewide/2010/tl_2012_36_vtd10.zip
 		-s_srs EPSG:4269 -t_srs EPSG:4326 -nln state -append -f GPKG $@ \
 		'/vsizip/data/36-new-york/statewide/2010/tl_2012_36_vtd10.zip/tl_2012_36_vtd10.shp'
 
+	# Add 001-albany
+	ogr2ogr -sql "SELECT '2012' AS year, '36' AS state, '001' AS county, CONCAT('36001', CAST(DistName AS character(20))) AS precinct, 'polygon' AS accuracy FROM \"2012_EDs-Albany\"" \
+		-s_srs EPSG:2260 -t_srs EPSG:4326 -nln state -append -f GPKG $@ \
+		'data/36-new-york/001-albany/2012/2012_EDs-Albany.shp'
+
+	# Add 003-allegany
+	ogr2ogr -sql "SELECT '2012' AS year, '36' AS state, '003' AS county, CONCAT('36003', CAST(NAMELSAD10 AS character(20))) AS precinct, 'polygon' AS accuracy FROM ElectionDistricts2012" \
+		-s_srs EPSG:2260 -t_srs EPSG:4326 -nln state -append -f GPKG $@ \
+		'data/36-new-york/003-allegany/2012/ElectionDistricts2012.shp'
+	ogr2ogr -sql "SELECT '2014' AS year, '36' AS state, '003' AS county, CONCAT('36003', CAST(NAMELSAD10 AS character(20))) AS precinct, 'polygon' AS accuracy FROM ElectionDistricts2014" \
+		-s_srs EPSG:2260 -t_srs EPSG:4326 -nln state -append -f GPKG $@ \
+		'data/36-new-york/003-allegany/2014/ElectionDistricts2014.shp'
+
+	# Add 007-broome... appears to be 2016 (date modifieds of files = 2016)
+	ogr2ogr -sql "SELECT '2016' AS year, '36' AS state, '007' AS county, CONCAT('36007', CAST(NAME AS character(20))) AS precinct, 'polygon' AS accuracy FROM broome_ed" \
+		-s_srs EPSG:2260 -t_srs EPSG:4326 -nln state -append -f GPKG $@ \
+		'data/36-new-york/007-broome/2016/broome_ed.shp'
+
+	# Add 009-cattaraugus
+	ogr2ogr -sql "SELECT '2012' AS year, '36' AS state, '009' AS county, CONCAT('36009', CAST(DISTRICT AS character(20))) AS precinct, 'polygon' AS accuracy FROM CattaraugusVotingDistricts" \
+		-s_srs EPSG:2260 -t_srs EPSG:4326 -nln state -append -f GPKG $@ \
+		'data/36-new-york/009-cattaraugus/2012/CattaraugusVotingDistricts.shp'
+
+	# Add 011-cayuga
+	ogr2ogr -sql "SELECT '2012' AS year, '36' AS state, '011' AS county, CONCAT('36011', CAST(MUNI AS character(20)), CAST(DISTRICT AS character(20))) AS precinct, 'polygon' AS accuracy FROM Election_Districts" \
+		-s_srs EPSG:2260 -t_srs EPSG:4326 -nln state -append -f GPKG $@ \
+		'data/36-new-york/011-cayuga/2012/Election_Districts.shp'
+	# This one errors with: "Failed to reproject feature 38 (geometry probably out of source or destination SRS)."
```

Comment:
Give this a try?

- https://gis.stackexchange.com/questions/14071/what-is-the-intended-behavior-of-skipfailures-in-ogr2ogr

Reply:
That seems to work! It still reports the failure, but presumably (I haven't yet tried the full Makefile run locally) still adds the bulk/rest of that shapefile's layer to the map